src/plan_expand_hypertable.c

```diff
@@ -95,6 +95,24 @@ is_time_bucket_function(Expr *node)
 	return false;
 }
 
+#if PG13_GE
+/* PG13 merged setup_append_rel_array with setup_simple_rel_arrays */
+static void
+setup_append_rel_array(PlannerInfo *root)
```

Comment:
I don't see this function called anywhere in this PR so I don't understand its purpose.

Reply:
setup_append_rel_array is a postgres function that got removed in PG13